lib/internal/process.js

```diff
@@ -65,6 +65,13 @@ function setupConfig(_source) {
     if (value === 'false') return false;
     return value;
   });
+  if ( process.config.variables.v8_enable_i18n_support && global.hasOwnProperty('Intl') 
+        && Intl.hasOwnProperty('v8BreakIterator') && process.config.variables.icu_small 
+        && !process.icu_data_dir ) {
+      Intl.v8BreakIterator = function v8BreakIterator() { 
```

Comment:
This should be installed with the same property descriptor as the original v8BreakIterator, e.g.:
```js
const des = Object.getOwnPropertyDescriptor(Intl, 'v8BreakIterator');
des.value = function v8BreakIterator() { /* ... */ };
Object.defineProperty(Intl, 'v8BreakIterator', des);
```
You don't have to use this exact code but the property should at least be non-enumerable.